ledger-service/http-json-cli/src/main/scala/com/daml/http/Config.scala

```diff
@@ -129,21 +129,21 @@ private[http] object JdbcConfig
       s"${indent}url -- JDBC connection URL,\n" +
       s"${indent}user -- database user name,\n" +
       s"${indent}password -- database user password,\n" +
-      s"${indent}createSchema -- boolean flag, if set to true, the process will re-create database schema and terminate immediately.\n" +
+      s"${indent}schemaHandling -- option setting how the schema should be handled. Valid options are ForceCreateAndTerminate, CheckAndTerminateIfWrong, CreateOrUpdateAndContinue, ForceCreateAndContinue.\n" +
```

Comment:
this breaks backwards compatibility this is no good.

Reply:
I deprecated the old flag (so it's still there but advised to use the new one) and adjusted the info with cf1632f